op25/gr-op25_repeater/www/www-static/main.js

```diff
@@ -640,13 +1211,35 @@ function f_tune_button(command) {
     }
 }
 
+function toggle_js_plot_display(command) {
```

Comment:
This logic is super buggy and can get out of sync if the server decides to stop sending graph data updates. It should be re-worked.